### DISCUSSION TITLE
Request for Direct Numeric Input for Adjustments in Your Software

### DISCUSSION THREAD

**Discussion by micsama** [rxhanson/Rectangle#1147] [Ideas]
Hello,

I have been trying out your software and find it very useful. However, I've encountered an inconvenience that I would like to bring to your attention. As shown in the red box in the attached image, when I need to make adjustments in this area, I can only do so by slowly dragging the mouse. It would be much more user-friendly if I could directly edit the numbers on the right side to make these adjustments.

I believe that implementing this feature would improve the overall usability of the program in other areas as well. Thank you for considering my suggestion!

Best regards,
<img width="490" alt="image" src="https://user-images.githubusercontent.com/41811005/236772028-c434e757-feb7-4c5c-980e-6f194c6948c6.png">

**Comment by rxhanson** [comment]
Thanks for the feedback. I don't have the bandwidth to add it in myself, but would welcome a good pull request with this change.
